app/views/hq/orphans/_form.html.erb

```diff
@@ -0,0 +1,224 @@
+<%= form_for [:hq, orphan] do |f| %>
+  <%= render 'shared/errors', object: f.object %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Orphan Details</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :name %>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :date_of_birth %>
+        <%= f.text_field :date_of_birth, class: "form-control date-picker", value: orphan.date_of_birth %>
+      </div>
+      <div class="form-group">
+        <%= f.label :gender %>
+        <%= f.select :gender,
+          Settings.lookup.gender, {}, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :health_status %>
+        <%= f.text_field :health_status, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :schooling_status %>
+        <%= f.text_field :schooling_status, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :goes_to_school %>
+        <%= f.check_box :goes_to_school %> 
+      </div>
+      <div class="form-group">
+        <%= f.label :orphan_status_id %>
+        <%= f.collection_select :orphan_status_id,
+          orphan_statuses, :id, :name, {}, 
+          {class: "form-control"} %>
+      </div>
+      <div class="form-group">
+        <%= f.label :orphan_sponsorship_status_id %>
+        <%= f.collection_select :orphan_sponsorship_status_id,
+          orphan_sponsorship_statuses, :id, :name, {}, 
+          {class: "form-control", disabled: "true"} %>
+      </div>
+      <div class="form-group">
+        <%= f.label :priority %>
+        <%= f.select :priority, 
+          %w(Normal High), {}, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :created_at %>
+        <%= f.text_field :created_at, class: "form-control date-picker", value: orphan.created_at %>
+      </div>
+      <div class="form-group">
+        <%= f.label :updated_at %>
+        <%= f.text_field :updated_at, class: "form-control date-picker", value: orphan.updated_at %>
```

Comment:
I don't think we want to allow manual updating of `created_at` & `updated_at`. Displaying them is probably useful (although they are not in AA and the client has not complained), but they should not be enabled, nor should they be included in the list of permitted params. It would also be good to give them more user-friendly and OSRA-specific labels, such as "Record imported on" and "Record last updated on".